.speakeasy/workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflowVersion: 1.0.0
+sources:
+    openapi:
+        inputs:
+            - location: ../../apps/web/openapi.json
```

Comment:
Same input from root directory

Reply:
Thats right this in the `apps` root directory